charts/atlantis/values.yaml

```diff
@@ -172,6 +172,10 @@ serviceAccountSecrets: {}
 # Default values for atlantis (override as needed).
 ## -------------------------- ##
 
+# -- When set, ignores the port specified by the Service's targetPort attribute.
+# This is relevant when another container is injected into the same Pod.
+containerPort:
```

Comment:
This field should be of type integer alone and when it’s the default type value of zero it wouldn’t be used.